x-pack/metricbeat/include/list.go

```diff
@@ -11,6 +11,7 @@ import (
 	_ "github.com/elastic/beats/x-pack/metricbeat/module/aws"
 	_ "github.com/elastic/beats/x-pack/metricbeat/module/aws/cloudwatch"
 	_ "github.com/elastic/beats/x-pack/metricbeat/module/aws/ec2"
+	_ "github.com/elastic/beats/x-pack/metricbeat/module/aws/elb"
```

Comment:
In principle light metricsets shouldn't appear here as they don't have code, this one appears here probably because of the `elb.go` file, that I think should be moved to `elb_test.go`.